tests/test-slirp4netns-configure.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xeuo pipefail
+
+. $(dirname $0)/common.sh
+
+unshare -r -n sleep 1000 &
+child=$!
+
+wait_for_network_namespace $child
+
+slirp4netns -c $child tun11 &
```

Comment:
`11`?

Reply:
just a random number, so that is less likely that the device was configured by some other tool

Reply:
![image](https://user-images.githubusercontent.com/9248427/43301024-8b9de366-919d-11e8-8944-444a33999457.png)